commit a940bf9e30d6e57ff6b13003d371a9ba4c43ae96
Author: Steve Dickson <steved@redhat.com>
Date:   Thu Jun 5 09:40:23 2014 -0400

    xdr_rejected_reply: Don't crash with invalid server rejection
    
    It seems legacy servers like to return invalid
    rejection status when a RPC is rejected. This
    should not crash the app then it happens.
    
    Signed-off-by: Steve Dickson <steved@redhat.com>

diff --git a/src/rpc_prot.c b/src/rpc_prot.c
index e562deb..5841e51 100644
--- a/src/rpc_prot.c
+++ b/src/rpc_prot.c
@@ -150,7 +150,6 @@ xdr_rejected_reply(xdrs, rr)
 		return (xdr_enum(xdrs, (enum_t *)&(rr->rj_why)));
 	}
 	/* NOTREACHED */
-	assert(0);
 	return (FALSE);
 }
 
